hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -93,6 +94,23 @@ public void setBlockDeletionInterval(Duration duration) {
     this.blockDeletionInterval = duration.toMillis();
   }
 
+  @Config(key = "block.deleting.limit.per.interval",
+      defaultValue = "1000",
```

Comment:
Let's increase the default value to 5000.
Based on the perf test 1 million blocks deleted in ~60 mins for 10 node cluster.
10^6 / (3 * 5000) = ~66 mins. (3 - numPipelines).

Reply:
Increased the default value to 5000.